src/js/app.js

```diff
@@ -0,0 +1,61 @@
+$(function() {
+  $('.limiter').hide();
+
+  $('form').on('submit', function(e){
+    e.preventDefault();
+    getResults($('#inputform').val());
+    $('#inputform').val('');
+    $('.container').hide();
+    $('.limiter').show();
+  })
+
+  $('#back-btn').click(function() {
+    location.reload();
```

Comment:
Note that whenever we do a hard reload, we have to go fetch the page again, either from the server or cache. Are there some ways that we could be more efficient by refreshing the page? Note that you're hiding and showing elements using jQuery on form submission.